docs/sdks/architecture/deployment-modes.md

```diff
@@ -0,0 +1,143 @@
+# Deployment modes
+
+Eppo's architecture is highly composable and can support a variety of deployment modes depending on what best fits your stack. This page describes a few common patterns for using Eppo, some tradeoffs, and a few anti-patterns to avoid (or at least be aware of).
+
+## Considerations when integrating a flagging service
+
+When weighing the pros and cons of the approaches outlined below, there are several dimension to consider:
+
+1. **Ease of integration and maintenance** - Do you want to use a system out of the box to minimize integration and operational overhead? Or are you comfortable maintaining some additional code on top of the service?
+2. **Performance and latency** - What is your tolerance for load times for flagging and experiment configurations?
+3. **Reliability and redundancy** - How many layers of redundancy do you want to have to ensure user experiences are never interrupted?
+4. **Quality of data capture** - How well does a given deployment pattern guarantee that the data captured will lead to high-quality decision making? What level of confidence do you need that a given architecture will avoid data quality issues like Sample Ratio Mismatch (SRM)?
+
+Each company might weigh each of these consideration differently. The purpose of this page is to highlight common patterns across Eppo customers and discuss the tradeoffs on the dimensions above.
+
+## Common deployment patterns 
+
+### Local flag evaluation using configurations from CDN (recommended)
+
+The simplest and least error prone approach is to load feature flag configurations from Eppo's CDN each time the SDK is initialized (with optional subsequent polling for updates). This only requires one network call to Eppo's CDN at app initialization, which typically returns in under 15ms. 
```

Comment:
Should there be additional information about how the config file works? What does the client get from Eppo in the network call at this point?

Reply:
Yes I think so. I'm not sure if this is the page for it though. I'm hoping to write up another page on what exactly is in the configuration file soon